api_examples/graphql/queries/aws_account_import.graphql

```diff
@@ -0,0 +1,122 @@
+# Description
+# -----------
+#
+# The mutation query import AWS account to a turbot folder. The mutation query works as in 2 step.
```

Comment:
The mutation query works as in 2 step.

Better English.
The mutation query needs to be run in 2 steps.